roles/agnosticv/templates/catalog_item.yml.j2

```diff
@@ -1,50 +1,19 @@
 ---
-apiVersion: template.openshift.io/v1
-kind: templates
+apiVersion: babylon.gpte.redhat.com/v1
+kind: CatalogItem
 metadata:
-  name: {{ _name | to_json }}
   annotations:
-    description: {{ merged_vars.__meta__.catalog.description | d(default_description) | to_json }}
-    openshift.io/display-name: {{ merged_vars.__meta__.catalog.display_name| default(_name) | to_json }}
-    openshift.io/long-description: {{ merged_vars.__meta__.catalog.long_description | d('') | to_json }}
-    tags: quickstart,babylon,{{ account | replace('_', '-') }},{{ merged_vars.__meta__.catalog.tags |  default([]) | join(',') }}
-    template.openshift.io/bindable: "false"
+    babylon.gpte.redhat.com/description: {{ _catalog.description | default(default_description) | to_json }}
+    babylon.gpte.redhat.com/displayName: {{ _catalog.display_name | default(_name) | to_json }}
+    babylon.gpte.redhat.com/icon: {{ _catalog.icon | default('') | to_json }}
+    babylon.gpte.redhat.com/keywords: {{ _catalog.keywords | default([]) | join(',') | to_json }}
+  labels: {{ default_labels | combine(_catalog.labels | d({})) | to_json }}
```

Comment:
I think for labels we will want to make it a bit easier for our users...
We are prefixing our labels to match best practices...
```
    labels:
      babylon.gpte.redhat.com/Category: Labs
      babylon.gpte.redhat.com/OpenShift_Version: "4.7"
      babylon.gpte.redhat.com/Product: OpenShift
      babylon.gpte.redhat.com/Provider: GPTE
```
But in the catalog our users probably should just set:
```
    labels:
      Category: Labs
      OpenShift_Version: "4.7"
      Product: OpenShift
      Provider: GPTE
```